g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/login/LTSSocketFactory.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019-2022 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vip.core.login;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.security.SecureRandom;
+
+import javax.net.SocketFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.TrustManager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LTSSocketFactory extends SSLSocketFactory {
+	private static Logger LOGGER = LoggerFactory.getLogger(LTSSocketFactory.class);
+	private SSLSocketFactory socketFactory;
+
+	public LTSSocketFactory() {
+		try {
+			SSLContext ctx = SSLContext.getInstance("TLS");
+
+			ctx.init(null, new TrustManager[] { new LTSTrustmanager() }, new SecureRandom());
```

Comment:
Please  check this scan faulure.

Reply:
we only use the server side https validation. this codes is client side https validation